flume/README.md

```diff
@@ -342,6 +342,8 @@ The typical configuration when using the `HTTPSource`, the `OrionRestHandler`, t
 
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/af19dc77924e4d15f2d0dfc86ac8d0ce2cf30c63